src/MailHide.php

```diff
@@ -186,15 +186,45 @@ public function getDefaultOptions()
      */
     public function setPrivateKey($privateKey)
     {
-        parent::setPrivateKey($privateKey);
+        parent::setSecretKey($privateKey);
 
         /* Pack the private key into a binary string */
-        $this->privateKeyPacked = pack('H*', $this->privateKey);
+        $this->privateKeyPacked = pack('H*', $this->getSecretKey());
 
         return $this;
     }
 
     /**
+     * get the private key property
+     *
+     * @return string
+     */
+    public function getPrivateKey()
+    {
+        return parent::getSecretKey();
+    }
+
+    /**
+     * set the public key property
+     *
+     * @param string $publicKey
+     */
+    public function setPublicKey($publicKey)
+    {
+        return parent::setSiteKey($publicKey);
+    }
+
+    /**
+     * Get the public key property
+     *
+     * @return string
+     */
+    public function getPublicKey()
+    {
+        return parent::getSiteKey();
+    }
```

Comment:
They aren't actually deprecated for MailHide as that's the correct wording for its API.